src/pages/Delivery/Delivery.styled.jsx

```diff
@@ -71,20 +73,21 @@ const ChatBtn = styled.img`
 const HeaderTitle = styled.div`
   display: flex;
   text-align: left;
-  margin: 0 auto;
-  font-weight: 900;
-  line-height: 17px;
+  justify-content: center ;
+  font-weight: 700;
+  line-height: 57px;
   font-family: 'Poppins', sans-serif;
 `
 const HeaderSubTitle = styled.div`
   display: flex;
   text-align: left;
+  justify-content: center ;
```

Comment:
Check indentation or try to use npm run eslint or npm run lint

Reply:
I think this one is caught by stylelint, for eslint or prettier is "just some random string"